packages/manager/apps/pci-volume-snapshot/src/pages/listing/Listing.page.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+  FilterCategories,
+  FilterTypeCategories,
+  FilterComparator,
+} from '@ovh-ux/manager-core-api';
+import { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDatagridColumn } from '@/pages/listing/useDatagridColumn';
+import { usePaginatedVolumeSnapshot } from '@/api/hooks/useSnapshots';
+
+export default function ListingPage() {
+  const { t } = useTranslation(['volumes']);
+
+  const { projectId } = useParams();
+  const hrefProject = useProjectUrl('public-cloud');
+  const { data: project } = useProject();
+
+  const { hasMaintenance, maintenanceURL } = useProductMaintenance(
+    projectId || '',
+  );
+  const columns = useDatagridColumn();
+  const [searchField, setSearchField] = useState('');
+  const { filters, addFilter, removeFilter } = useColumnFilters();
+  const { pagination, setPagination, sorting, setSorting } = useDataGrid({
+    id: 'creationDate',
+    desc: true,
+  });
+  const { paginatedSnapshots, isLoading } = usePaginatedVolumeSnapshot(
+    projectId || '',
+    pagination,
+    sorting,
+    filters,
+  );
+
+  const onHandleSearch = () => {
+    setPagination({
+      pageIndex: 0,
+      pageSize: pagination.pageSize,
+    });
+    addFilter({
+      key: 'name',
+      value: searchField.trim(),
+      comparator: FilterComparator.Includes,
+      label: '',
+    });
+    setSearchField('');
+  };
+
+  return (
+    <RedirectionGuard condition={false} isLoading={false} route={''}>
+      <BaseLayout
+        breadcrumb={
+          <OdsBreadcrumb>
+            <OdsBreadcrumbItem
+              label={project?.description}
+              href={hrefProject}
+            />
+            <OdsBreadcrumbItem
+              label={t('pci_projects_project_storages_snapshots_title')}
+              href={'#'}
+            />
+          </OdsBreadcrumb>
+        }
+        header={{
+          title: t('pci_projects_project_storages_snapshots_title'),
+          headerButton: <PciGuidesHeader category={'storage'} />,
+        }}
+      >
+        <div className="flex flex-col gap-5">
+          <div>
+            <Notifications />
+            {hasMaintenance && (
+              <PciMaintenanceBanner
+                maintenanceURL={maintenanceURL}
+                productName={t('pci_projects_project_storages_snapshots_title')}
+              />
+            )}
+            <PciAnnouncementBanner projectId={projectId} />
+          </div>
+          <div className="sm:flex items-center justify-between">
+            <div className="sm:flex">
+              <OdsButton
+                size="sm"
+                color="primary"
+                variant="outline"
+                className="xs:mb-0.5 sm:mb-0"
+                icon="plus"
+                label={t('pci_projects_project_storages_snapshots_add_label')}
+                onClick={() => {
+                  window.location.href = `${hrefProject}/storages/volume-backup/create`;
```

Comment:
for external links (links between apps) you can use the shell navigation plugin, exemple of usage here : https://github.com/ovh/manager/blob/master/packages/manager/apps/veeam-backup/src/components/Breadcrumb/Breadcrumb.tsx#L46

since the app is displayed in the container, the correct way to naviage to an external url is to use window.top.location (you don't have to care about this if you are using the shell plugin)